package.json

```diff
@@ -53,10 +53,12 @@
 		"@aws-sdk/credential-providers": "^3.817.0",
 		"@inquirer/search": "^3.0.15",
 		"@openrouter/ai-sdk-provider": "^0.4.5",
+		"@streamparser/json": "^0.0.22",
```

Comment:
is this being used for the CLI exclusively ? wondering how this is being used, only able to see it being used in the tests